commands/import.go

```diff
@@ -0,0 +1,251 @@
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/spf13/cobra"
+)
+
+const fakeUserID = "nouser"
+
+var ImportCmd = &cobra.Command{
+	Use:   "import",
+	Short: "Management of imports",
+}
+
+var ImportUploadCmd = &cobra.Command{
+	Use:     "upload [filepath]",
+	Short:   "Upload import files",
+	Example: " import upload import_file.zip",
+	Args:    cobra.ExactArgs(1),
+	RunE:    withClient(importUploadCmdF),
+}
+
+var ImportListCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List import files",
+}
+
+var ImportListAvailableCmd = &cobra.Command{
+	Use:     "available",
+	Short:   "List available import files",
+	Example: " import list available",
+	Args:    cobra.NoArgs,
+	RunE:    withClient(importListAvailableCmdF),
+}
+
+var ImportListIncompleteCmd = &cobra.Command{
+	Use:     "incomplete",
+	Short:   "List incomplete import files uploads",
+	Example: " import list incomplete",
+	Args:    cobra.NoArgs,
+	RunE:    withClient(importListIncompleteCmdF),
+}
+
+var ImportListJobsCmd = &cobra.Command{
+	Use:     "jobs [importJobID]",
+	Example: " import list jobs",
+	Short:   "List import jobs",
+	Args:    cobra.MaximumNArgs(1),
+	RunE:    withClient(importListJobsCmdF),
+}
+
+var ImportProcessCmd = &cobra.Command{
+	Use:     "process [importname]",
+	Example: " import process 35uy6cwrqfnhdx3genrhqqznxc_import.zip",
+	Short:   "Start an import job",
+	Args:    cobra.ExactArgs(1),
+	RunE:    withClient(importProcessCmdF),
+}
+
+func init() {
+	ImportUploadCmd.Flags().Bool("resume", false, "Set to true to resume an incomplete import upload.")
+	ImportUploadCmd.Flags().String("upload", "", "The ID of the import upload to resume.")
+	ImportListJobsCmd.Flags().Int("limit", 10, "The maximum number of jobs to show.")
+	ImportListCmd.AddCommand(
+		ImportListAvailableCmd,
+		ImportListIncompleteCmd,
+		ImportListJobsCmd,
+	)
+	ImportCmd.AddCommand(
+		ImportUploadCmd,
+		ImportListCmd,
+		ImportProcessCmd,
+	)
+	RootCmd.AddCommand(ImportCmd)
+}
+
+func importListIncompleteCmdF(c client.Client, command *cobra.Command, args []string) error {
+	isLocal, _ := command.Flags().GetBool("local")
+	userID := "me"
+	if isLocal {
+		userID = fakeUserID
+	}
+
+	uploads, resp := c.GetUploadsForUser(userID)
+	if resp.Error != nil {
+		return fmt.Errorf("failed to get uploads: %w", resp.Error)
+	}
+
+	var imports []*model.UploadSession
+	for i := range uploads {
+		if uploads[i].Type == model.UploadTypeImport {
+			imports = append(imports, uploads[i])
+		}
+	}
+
+	if len(imports) == 0 {
+		printer.Print("No incomplete import uploads found")
+		return nil
+	}
+
+	for _, us := range imports {
+		completedPct := float64(us.FileOffset) / float64(us.FileSize) * 100
+		printer.PrintT(fmt.Sprintf("  ID: {{.Id}}\n  Name: {{.Filename}}\n  Uploaded: {{.FileOffset}}/{{.FileSize}} (%0.0f%%)\n", completedPct), us)
+	}
+
```

Comment:
What do you think about printing the total number every time? It would save us from one loop :)

PS: this comment applies to similar occurrences.